### PR TITLE
ci: additional workflows and update template

### LINKED ISSUE #608: Update workflow settings to be like the hub
- [ ] e2e tests
- [ ] linters
- [ ] etc..

### PR DESCRIPTION
**Pull request by hoangdv2429**
# PR Standards

## Opening a pull request should be able to meet the following requirements

Close #608

---

For Author:

- [x]  Targeted PR against correct branch
- [x]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [x]  Linked to Github issue with discussion and accepted design
- [x]  Targets only one github issue
- [ ]  Wrote unit and integration tests
- [ ]  All CI checks have passed
- [ ]  Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code)

---

For Reviewer:

- [ ]  confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Reviewers assigned
- [ ]  confirmed all author checklist items have been addressed

---

After reviewer approval:

- [ ]  In case targets main branch, PR should be squashed and merged.
- [ ]  In case PR targets a release branch, PR should be rebased.

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -51,4 +51,4 @@ jobs:
         uses: nosborn/github-action-markdown-cli@v3.3.0
         with:
           files: .
-          config-file: .markdownlint.yaml
\ No newline at end of file
+          config-file: .markdownlint.yaml
```
Comment (mtsitrin) [inline .github/workflows/lint.yml:54]: isn't dup of `[markdown_lint.yml]`?

**Review by mtsitrin — COMMENTED** [review]
linter fails

**Comment by hoangdv2429** [comment]
> linter fails

Currently I can't see those errors on my computer with `golangci-lint run`. So fixing those errors need a little bit more time.

**Comment by omritoptix** [comment]
Those are the errors @hoangdv2429 

Screenshot 2024-03-24 at 19 55 38

**Comment by hoangdv2429** [comment]
@omritoptix @mtsitrin There is an error in dymint with getNodeConfig, do you want me to fix it in this PR or create another one for it ?

File: .github/workflows/lint.yml (hunk 1 above)
Comment (hoangdv2429) [reply .github/workflows/lint.yml]: I removed the old one

**Pull request merged**